packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -697,15 +697,7 @@ class _PopupMenuRouteLayout extends SingleChildLayoutDelegate {
 
     final double buttonHeight = size.height - position.top - position.bottom;
     // Find the ideal vertical position.
-    double y = position.top;
-    if (selectedItemIndex != null && itemSizes != null) {
-      double selectedItemOffset = _kMenuVerticalPadding;
-      for (int index = 0; index < selectedItemIndex!; index += 1) {
-        selectedItemOffset += itemSizes[index]!.height;
-      }
-      selectedItemOffset += itemSizes[selectedItemIndex!]!.height / 2;
-      y = y + buttonHeight / 2.0 - selectedItemOffset;
-    }
+    final double y = position.top + buttonHeight;
```

Comment:
With this change, would the menu still show up below the button even if the `position` is set to `PopupMenuPosition.over`?

Reply:
> With this change, would the menu still show up below the button even if the position is set to PopupMenuPosition.over?

No, not unless an Offset is provided.

If the position attribute is set to PopupMenuPosition.over, the menu will be drawn above it, if it is set to PopupMenuPosition.under, it will be drawn under it.

https://user-images.githubusercontent.com/73116038/205382980-217b0b25-2299-4535-9149-1ad1f4ee4761.mp4